runtime/src/launch/slurm-srun/launch-slurm-srun.c

```diff
@@ -156,6 +156,23 @@ static int getCoresPerLocale(int nomultithread, int32_t localesPerNode) {
   }
   return numCores / localesPerNode;
 }
+
+
+static chpl_bool getSlurmDebug(chpl_bool batch) {
+  chpl_bool result = false;
+  char *debugString = getenv("SLURM_DEBUG");
+  if (debugString) {
+    result = (atoi(debugString) != 0) ? true : false;
+  } else if (batch) {
+    debugString = getenv("SBATCH_DEBUG");
+    if (debugString) {
+      result = (atoi(debugString) != 0) ? true : false;
+    }
+  }
+  return result;
+}
+
+
```

Comment:
I _think_ that `SLURM_DEBUG` only applies to `srun` so it's not intuitive to me that setting it would impact what we do for `sbatch`.

Reply:
I think you are correct, I guess I misread the man pages. I'll make the fix.